src/main/java/com/google/sps/data/SmallCityService.java

```diff
@@ -43,6 +43,7 @@ public SmallCityService() { }
   **/
   public void createUserWithZip(String zipCode) {
     this.user = new User(zipCode);
+    LOGGER.info(this.user.getGeolocation().lat + ", " + this.user.getGeolocation().lng);
```

Comment:
Leftover bits from debugging? Looks like this can be removed.

Reply:
Removed it!